blobfuse2-release.yaml

```diff
@@ -1356,6 +1356,122 @@ stages:
               temp_dir: $(temp_dir)
               container: $(container)
 
+      - job: Set_9
+        timeoutInMinutes: 120
+        strategy:
+          matrix:
+            Rocky-8.0:
+              agentName: "blobfuse-rocky8"
```

Comment:
Getting error while publishing to Rocky Linux. Holding this until the issue is resolved.

Reply:
Fixed